bin/sdk/gutenberg.js

```diff
@@ -41,8 +40,9 @@ exports.config = ( { argv: { inputDir, outputDir }, getBaseConfig } ) => {
 	let presetBetaBlocks;
 
 	if ( fs.existsSync( presetPath ) ) {
-		presetBlocks = require( presetPath );
-		presetBetaBlocks = fs.existsSync( presetBetaPath ) ? require( presetBetaPath ) : [];
+		const presetIndex = require( presetPath );
+		presetBlocks = presetIndex.production ? presetIndex.production : [];
+		presetBetaBlocks = presetIndex.beta ? presetIndex.beta : [];
```

Comment:
Would this shorthand work here `presetBetaBlocks = presetIndex.beta || []` ?

Reply:
Seems like a use case for lodash's `get( presetIndex, [ 'beta' ], [] )`